ARTICLE-7.markdown

```diff
@@ -13,7 +13,7 @@ Member-Owners.
 
 "Profit" shall be defined as the excess of revenues and gains
 over expenses and losses for a fiscal year attributed to
-non-Member-Owners.
+non-Member Collaborators.
```

Comment:
This document explains the basics of Worker Coop accounting, @sapraaman :  http://ica-group.org/wp-content/uploads/2015/04/Internal-Capital-Accounts.pdf

The only change here is to be consistent with the term non-Member Collaborator as defined in Section I